### ISSUE TITLE
Field Transfer from new field to sales header

### ISSUE THREAD

**Issue by bpulido76** [hougaard/Simple-Object-Designer#64]
Hi. I created a new field on the customer table and would like to have the field transfer over to the sales header with the new field. Could this be done? Thank you.
![2022-04-04 10_35_12-View - Simple Object Designer - Dynamics 365 Business Central](https://user-images.githubusercontent.com/98491495/161567467-396820e7-d5d1-488b-bb64-207cd543b5ad.png)
![2022-04-04 10_36_26-View - Simple Object Designer - Dynamics 365 Business Central](https://user-images.githubusercontent.com/98491495/161567620-93536e6f-0486-4a6f-9909-56b36721cb13.png)
?

**Comment by hougaard** [comment]
Yes, just click on "Field Transfers" from the customer field and select sales header.

**Comment by hougaard** [comment]
... Yes, you need to decide if it's the sell-to or bill-to that triggers the transfer.

**Comment by hougaard** [comment]
The "Via field" is the field that will trigger the transfer, not the destination.
